src/yb/consensus/raft_consensus.cc

```diff
@@ -1506,6 +1506,14 @@ Status RaftConsensus::Update(
     return STATUS(IllegalState, "Rejected: --TEST_follower_reject_update_consensus_requests "
                                 "is set to true.");
   }
+  auto wait_state = util::WaitStateInfo::CurrentWaitState();
+  if (wait_state) {
+    wait_state->UpdateMetadata(
+      yb::util::AUHMetadata{
+          .top_level_node_id = "consensus",
+          .top_level_request_id = {0, 1},
+          .query_id = -1});
+  }
```

Comment:
IMO top level node id should be the actual node id, and we can reserve some top level request ids for special kind of requests like these, e.g. `{0, 0}` will be for flush and compaction, `{0, 1}` for consensus.

Also, where are the client node ip and request id being set?

Reply:
Consensus ops are not specifically tied to any specific query. The consensus rpc's get merged for different queries, besides we don't actually persist the AUH/metadata into the WAL logs, so the context is lost when entering the consensus layer.

So I don't think top level node id per se makes a lot of sense -- the way we interpret it for other "queries".

Just trying to populate it with something, so that it stands out, as something non-empty.